packages/@lwc/rollup-plugin/src/index.js

```diff
@@ -99,14 +99,13 @@ module.exports = function rollupLwcCompiler(pluginOptions = {}) {
             }
 
             // If we don't find the moduleId, just resolve the module name/namespace
-            const moduleEntry = Object.values(modulePaths).find(r => id === r.entry);
```

Comment:
@apapko This returns sometimes `undefined`, and sometimes it returns:

```json
{
  entry: "path/to/entry.js",
  specifier: "x/app"
}
```
notice that there's no name or namespace, thus getting passed to `transform` as `undefined`. This was causing tests to fail when the options were being validated. Does that mean that `name` and `namespace` aren't really required?

I did notice that `getModuleQualifiedName` always returned the expected `name` and `namespace`, though. Thoughts?

Reply:
as of now, the name and namespace are only required for template transformer